SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security
+
+C2PA open source repos are maintained in partnership with Adobe. At this time, security policies are maintained solely by Adobe employees.
```

Comment:
This C2PA open-source library is maintained in partnership with Adobe. At this time, Adobe is taking point on accepting security reports through their HackerOne portal and public bug bounty program.